libdrgn/dwarf_index.h

```diff
@@ -319,6 +319,8 @@ struct drgn_error *drgn_dwarf_index_get_die(struct drgn_dwarf_index_die *die,
 					    Dwarf_Die *die_ret,
 					    uint64_t *bias_ret);
 
+bool find_definition(struct drgn_dwarf_index *dindex, uintptr_t die_addr,
```

Comment:
Since this is being made public, it should be renamed `drgn_dwarf_index_find_definition()` and given a comment.